### DISCUSSION TITLE
install.sh from archive fails

### DISCUSSION THREAD

**Discussion by AlexIova** [dmitry-s93/MControlCenter#64] [Q&A] [answered]
Sorry in advance if it's a stupid issue.

If one downloads the repository from github and tries to run `scripts/install.sh` as explained in the README, the error `cp: cannot stat './app/mcontrolcenter': No such file or directory` will occur and the installation will not proceed.

Is this a genuine bug or am I just getting something wrong?

**Comment by dmitry-s93** [comment, marked as answer]
Hello. To install from the archive, you need to download MControlCenter-x.x.x.tar.gz from the [releases](https://github.com/dmitry-s93/MControlCenter/releases) page.
